package.json

```diff
@@ -7,7 +7,9 @@
     "start": "npm run webpack-start",
     "webpack-start": "node scripts/environment.js && node scripts/generateManuals.js && webpack-dev-server",
     "rollup-start": "node scripts/environment.js && node scripts/generateManuals.js && rollup --environment SERVE:true -c --watch --no-treeshake",
-    "build": "node scripts/environment.js $RELEASE_NAME && node scripts/generateManuals.js && rm -rf dist/* && rollup --environment mode:production -c && scripts/create_versions.sh"
+    "build": "node scripts/environment.js $RELEASE_NAME && node scripts/generateManuals.js && rm -rf dist/* && rollup --environment mode:production -c && scripts/create_versions.sh",
+    "test": "mocha",
+    "test-watch": "mocha --watch"
```

Comment:
minor but instead of adding `test-watch` it can be done by adding `--` to `test`:
```
npm run test -- --watch
```
I do the same for the `lint` script:
```
npm run lint -- --fix
```